llarp/context.cpp

```diff
@@ -123,9 +132,9 @@ namespace llarp
     llarp_ev_loop_run_single_process(mainloop, logic);
     if (closeWaiter)
     {
-      // inform promise if called by CloseAsync
```

Comment:
any particular reason to remove this comment?

Reply:
no